app/views/users/settings.html.haml

```diff
@@ -0,0 +1,17 @@
+.row
+  .container
+    %h2
+      Settings
+      
+    - if current_user.email      
+      %h3
+        Email:
+        %strong
+          #{current_user.email}
+        
+    .btn-group
+      %a.btn.btn-default{ href: edit_user_path(current_user, change: "email")}
+        = current_user.email ? "Change Email" : "Set Email"
+        
+      %a.btn.btn-default{ href: edit_user_path(current_user, change: "password")}
```

Comment:
Hash attribute should end with one space before the closing brace<br>Line is too long. [81/80]